tfx/components/transform/component.py

```diff
@@ -108,12 +111,17 @@ def preprocessing_fn(inputs: Dict[Text, Any], custom_config:
         'preprocessing_fn'. See 'module_file' for expected signature of the
         function. Exactly one of 'module_file' or 'preprocessing_fn' must be
         supplied.
+      splits_config: A transform_pb2.SplitsConfig instance, providing splits
+        that should be analyzed and splits that should be transformed. Default
+        behavior (when splits_config is not set) is analyze the 'train' split
+        and transform all splits. If splits_config is set, analyze_splits cannot
+        be empty.
       transform_graph: Optional output 'TransformPath' channel for output of
         'tf.Transform', which includes an exported Tensorflow graph suitable for
         both training and serving;
       transformed_examples: Optional output 'ExamplesPath' channel for
         materialized transformed examples, which includes both 'train' and
-        'eval' splits.
+        'eval' splits if custom splits is not specified in splits_config.
```

Comment:
Should this mention what it should include if splits_config is specified? (similarly above for examples)

Reply:
i see, i updated the comments, thanks!

Reply:
i see, i updated the comments, thanks!